subgit/core.py

```diff
@@ -486,6 +489,108 @@ def pull(self, names):
                 log.info(f"Current commit summary on HEAD in git repo '{name}': ")
                 log.info(f"  {str(repo.head.commit.summary)}")
 
+    def delete(self, repo_names=None):
+        """
+        Method takes zero or one repo name as argument.
+
+        Checks if the repo(s) are valid git repo(s) and if there are any untracked changes.
+        """
+
+        config = self._get_config_file()
+
+        active_repos = self._get_active_repos(config)
+
+        active_repos_dict = config.get("repos", [])
+
+        if repo_names is None:  # If no names are specified in the command
+            repo_choices = ", ".join(active_repos_dict)
+
+        if isinstance(repo_names, list):  # If at least one name was specified
+            repo_choices = ", ".join(repo_names)
+
+        answer = self.yes_no(f"Are you sure you want to delete the following repos '{repo_choices}'")
+
+        if answer:
+            self._delete_repo(active_repos, repo_names)
+
+    @pysnooper.snoop()
+    def _delete_repo(self, active_repos, repos=None):
+        """
+        Helper method that recieves a list of repos to delete and takes care of the logic
+        """
+        if not repos:
+            repos = active_repos
+
+        in_conf_file = True
+
+        has_dirty_repos = False
+
+        repo_paths = []
+
+        bad_path = []
+
+        good_repos = []
+
+        for name in repos:
+            repo_paths.append(os.path.join(os.getcwd(), name))
+            if name not in active_repos:
+                log.warning(f"Argument: '{name}' does not exist in '.subgit.yml'")
+                in_conf_file = False
+
+        if not in_conf_file:
+            return 1
+
+        for path in repo_paths:
+            repo_name = basename(path)
+            if not os.path.exists(path):
+                log.warning(f"Path to repo does not exist: {repo_name} | Skipping {repo_name}")
+                bad_path.append(path)
+                repo_paths.remove(path)
+
+        for path in repo_paths:
+            repo_name = basename(path)
+            if not self.is_valid_repo(path):
+                log.warning(f"fatal: '{repo_name}' is not a git repo")
```

Comment:
The logging framework have several different logging levels, things that is breaking or errors or critical should get the correct logging level like

```suggestion
                log.critical(f"'{repo_name}' is not a git repo")
```